### PR TITLE
Spans indexed by intelligent retention filter can be viewed in Timeseries now

### PR DESCRIPTION
**Pull request by drichards-87**
### What does this PR do?
Fix outdated lines

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by drichards-87** [comment]
@cecile75 Git doesn’t appear to like the special character in your branch name, so I went ahead and renamed it.

File: content/en/tracing/trace_explorer/_index.md [hunk 1]
```diff
@@ -127,13 +127,15 @@ All spans indexed by custom retention filters *and* the intelligent retention fi
 {{% /tab %}}
 {{% tab "Timeseries View" %}}
 
-All spans indexed by custom retention filters (*not* the intelligent retention filter) are available to be searched when using trace analytics.
+All spans indexed by custom retention filters *and* the intelligent retention filter are available to be searched when using trace analytics.
 
-From the timeseries view, export your query to a [dashboard][1], a [monitor][2] or a [notebook][3] to investigate further or to alert automatically when an aggregate number of spans crosses a specific threshold.
+From the timeseries view, export your query to a [dashboard][1], a [monitor][2] or a [notebook][3] to investigate further or to alert automatically when an aggregate number of spans crosses a specific threshold. **Note**: Spans indexed by the intelligent retention filter are excluded from APM queries that appear in dashboards and notebooks and from trace analytics monitor evaluation. More information can be found [here][4].
```
Comment (drichards-87) [inline content/en/tracing/trace_explorer/_index.md:132]:
```suggestion
From the timeseries view, export your query to a [dashboard][1], a [monitor][2] or a [notebook][3] to investigate further or to alert automatically when an aggregate number of spans crosses a specific threshold. 

**Note**: Spans indexed by the intelligent retention filter are excluded from APM queries that appear in dashboards, notebooks, and from trace analytics monitor evaluations. For more information, see [Trace Retention][4].
```

File: content/en/tracing/trace_explorer/_index.md [hunk 2]
```diff
@@ -127,13 +127,15 @@ All spans indexed by custom retention filters *and* the intelligent retention fi
 {{% /tab %}}
 {{% tab "Timeseries View" %}}
 
-All spans indexed by custom retention filters (*not* the intelligent retention filter) are available to be searched when using trace analytics.
+All spans indexed by custom retention filters *and* the intelligent retention filter are available to be searched when using trace analytics.
```
Comment (drichards-87) [inline content/en/tracing/trace_explorer/_index.md:130]:
```suggestion
All spans indexed by custom retention filters **and** the intelligent retention filter are available to be searched when using trace analytics.
```

**Review by drichards-87 — COMMENTED** [review]
Left you some minor feedback from Docs.

**Review by drichards-87 — COMMENTED** [review]
Left you some minor feedback from Docs.

**Review by drichards-87 — COMMENTED** [review]
Left you some minor feedback from Docs.

File: content/en/tracing/trace_explorer/_index.md [hunk 3]
```diff
@@ -127,13 +127,17 @@ All spans indexed by custom retention filters *and* the intelligent retention fi
 {{% /tab %}}
 {{% tab "Timeseries View" %}}
 
-All spans indexed by custom retention filters (*not* the intelligent retention filter) are available to be searched when using trace analytics.
+All spans indexed by custom retention filters **and** the intelligent retention filter are available to be searched when using trace analytics.
```
Comment (cswatt) [inline content/en/tracing/trace_explorer/_index.md:130]:
```suggestion
All spans indexed by custom retention filters or the intelligent retention filter are available to be searched when using trace analytics.
```

OR means "any span indexed by either the custom retention filter or the intelligent retention filter is available"
AND means "only spans that have been indexed by both the custom retention filter and intelligent retention filter are available"

File: content/en/tracing/trace_explorer/_index.md [hunk 4]
```diff
@@ -127,13 +127,17 @@ All spans indexed by custom retention filters *and* the intelligent retention fi
...
 
-From the timeseries view, export your query to a [dashboard][1], a [monitor][2] or a [notebook][3] to investigate further or to alert automatically when an aggregate number of spans crosses a specific threshold.
+From the timeseries view, export your query to a [dashboard][1], a [monitor][2] or a [notebook][3] to investigate further or to alert automatically when an aggregate number of spans crosses a specific threshold.
```
Comment (cswatt) [inline content/en/tracing/trace_explorer/_index.md:132]:
```suggestion
From the timeseries view, export your query to a [dashboard][1], [monitor][2], or [notebook][3] to investigate further or to alert automatically when an aggregate number of spans crosses a specific threshold. 
```

**Review by cswatt — CHANGES_REQUESTED** [review]

**Review by cswatt — APPROVED** [review]

**Pull request merged**